pigeon.py

```diff
@@ -176,32 +189,15 @@ def is_crash_id(crash_id):
 
     """
     return (
-        # Verify length of the string
-        len(crash_id) == 36  # and
+        # Match structure
+        bool(CRASH_ID_RE.match(crash_id)) and
 
         # The 7-to-last character is a throttle result
-        # FIXME(willkg): We can re-enable this check later after
-        # -prod has been updated to use Antenna.
-        # crash_id[-7] in (ACCEPT, DEFER)
+        crash_id[-7] in (ACCEPT, DEFER)
     )
```

Comment:
This is the same validation we do in Socorro, so I'm copying it over.

Plus since Antenna has been the one-true collector, we always have ACCEPT or DEFER as the 7th-to-last index.